cypress/integration/26-register_publication/1409-creator_selects_publication_in_journal_and_peer_review_details_are_hidden.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Creator selects Publication in Journal and Peer Review Details are hidden
+
+    @1409
+    Scenario Outline: Creator selects Publication in Journal and Peer Review Details are hidden
+        Given that a Creator navigates to the Reference tab
+        And they select type Publication in Journal
+        When they select <Subtype>
+        Then they see that the Peer Review Details are hidden
+        Examples:
+            | Subtype              |
+            | Short communication  |
+            | Leader               |
+            | Letter to the editor |
+            | Review               |
```

Comment:
Dette Scenario er med i den andre PRen også, men der heter dette feltet "Book review"... Skulle denne PRen egentlig vært branchet ut fra den andre, eller skal denne merges inn i den i stedet for develop?

Reply:
Skulle branche ut fra develop, men har visst ikke gjort det, skal fikse det når jeg merger.